### PR TITLE
Prepare to ship FCS 39.0.0-preview

### PR DESCRIPTION
**Pull request by KevinRansom**
Prepare to ship fcs 39.0.0-preview

Reset FSharp.Core package version back to 5.0.0 because thats what shipped and set fcs version to 38.0.1

### CODE REVIEW

**Comment by baronfel** [comment]
I think the version number required us going to have to be 39, because there are a number of public surface area changes already in main from the 38 release.

I know that the reactor thread changes are in there, for example.

**Comment by baronfel** [comment]
Also, for this release could the MSBuild dependencies get added to the nuspec? It's a pretty prevalent gotcha that's hit several other consumers (including sharplab and xunit).

If that dependency gets extracted later then it can just be removed from the nuspec.

**Comment by KevinRansom** [comment]
@baronfel see: https://github.com/dotnet/fsharp/pull/10588

File: eng/Versions.props [hunk 1]
```diff
@@ -11,11 +11,11 @@
   </PropertyGroup>
   <!-- Version number computation -->
   <PropertyGroup>
-    <PreReleaseVersionLabel>beta</PreReleaseVersionLabel>
+    <PreReleaseVersionLabel>preview</PreReleaseVersionLabel>
```
Comment (cartermp) [inline eng/Versions.props:14]: is there a build number or something that comes after this?

Comment (KevinRansom) [reply eng/Versions.props:14]: No, the build number is computed by arcade, This just changes -beta- to -preview-, which is more accurate.

**Comment by KevinRansom** [comment]
@cartermp / @brettfo -- any issues or can we merge it?

File: eng/Versions.props (hunk 1 above)
Comment (brettfo) [reply eng/Versions.props:14]: This should remain `beta` since you're overriding it directly in FCS.fsproj.

**Review by brettfo — CHANGES_REQUESTED** [review]

File: eng/Versions.props (hunk 1 above)
Comment (KevinRansom) [reply eng/Versions.props]: fixed

**Review by brettfo — APPROVED** [review]

**Pull request merged**